src/LinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -84,6 +84,7 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
   void Visit(uint64_t event_timestamp, const StackSamplePerfEventData& event_data) override;
   void Visit(uint64_t event_timestamp, const CallchainSamplePerfEventData& event_data) override;
   void Visit(uint64_t event_timestamp, const UprobesPerfEventData& event_data) override;
+  void Visit(uint64_t event_timestamp, const UprobesWithStackPerfEventData& event_data) override;
```

Comment:
Same here regarding the order.

Reply:
Well, actually in the header it's probably better to follow the same order as in `PerfEventVisitor` so it can be left alone.